packages/core/admin/admin/src/content-manager/pages/EditSettingsView/components/FormModal.js

```diff
@@ -35,7 +35,7 @@ const FormModal = ({ onToggle, onMetaChange, onSizeChange, onSubmit, type }) =>
       return 'date';
     }
 
-    if (['decimal', 'float', 'integer', 'biginter'].includes(type)) {
+    if (['decimal', 'float', 'integer', 'biginteger'].includes(type)) {
```

Comment:
scary

Reply:
This one was ok it's just in order to display the right icon in the modal header.

Since you have spotted a bug maybe you could add some tests as well?